homeassistant/components/influxdb/__init__.py

```diff
@@ -285,7 +265,7 @@ def setup(hass, config):
         )
         event_helper.call_later(hass, RETRY_INTERVAL, lambda _: setup(hass, config))
         return True
-    except ApiException as exc:
+    except (ApiException, urllib3.exceptions.HTTPError) as exc:
```

Comment:
This is the only line of code I left in from the "unrelated to GLOB filtering" refactor work. The reason is because it should've been there to begin with, missing it was an oversight on my part when I did the Influx 2.0 work. My understanding of the library at the time was that it always returns an `ApiException` on error. This is not true, if the connection can't be made then it returns a `urllib3.exceptions.HTTPError`. Adding this line in makes V2 work just like V1, if a connection can't be made then it logs and error and retries after an wait interval.

Now I can change both of these so they return `False` on connection failure instead of retrying doing a wait to retry. But since the goal was to limit the changes to just things related to glob filtering I figured the smallest possible change was to fix the bug that prevented V2 from working like V1.